docs/configuration.md

```diff
@@ -1,5 +1,29 @@
 # Configuration
 
+The configuration is saved in the database, you can change and view them via the admin interface.
+
+You can find them in the appconfig table.
```

Comment:
This isn't needed, nobody should be checking the database for values they can see in settings.